packages/labs/test/selectTests.tsx

```diff
@@ -54,7 +53,8 @@ describe("<Select>", () => {
     });
 
     it("itemRenderer is called for each filtered child", () => {
-        select({}, "1999");
+        const wrapper = select({ query: "" });
+        wrapper.setProps({ query: "1999" });
```

Comment:
Modifying the query via a prop (i.e. controlling it) isn't necessarily going to test the same thing as modifying the query via `setState`. We should keep these uncontrolled tests uncontrolled and have explicit tests for controlled mode elsewhere in the file.